mne/inverse_sparse/mxne_inverse.py

```diff
@@ -364,15 +375,22 @@ def mixed_norm(evoked, forward, noise_cov, alpha, loose='auto', depth=0.8,
     .. footbibliography::
     """
     from scipy import linalg
-    if not (0. <= alpha < 100.):
+    if type(alpha) == int and not (0. <= alpha < 100.):
```

Comment:
```suggestion
    if not ("alpha" == "sure" or 0. <= alpha < 100.):
```

Reply:
should be no quotes around alpha in the suggested change